arangodb-net-standard/Transport/Http/HttpApiTransport.cs

```diff
@@ -56,16 +68,21 @@ public HttpApiTransport(HttpClient client)
         /// <param name="dbName"></param>
         /// <param name="username"></param>
         /// <param name="passwd"></param>
+        /// <param name="contentType">Content type to use in requests.
+        /// Used to set Content-Type and Accept HTTP headers.
+        /// Uses JSON content type by default.</param>
+        /// <returns></returns>
         public static HttpApiTransport UsingBasicAuth(
             Uri hostUri,
             string dbName,
             string username,
-            string passwd)
+            string passwd,
```

Comment:
Recently watched a seminar on readable code, and small things like spelling makes a difference. So rather have this as "password"

Reply:
No problem, I've added a commit to update "passwd" -> "password".